ersilia/core/tracking.py

```diff
@@ -4,6 +4,23 @@
 import os
 
 PERSISTENT_FILE_PATH = os.path.abspath("current_session.txt")
+# Temporary path to log files
+TEMP_FILE_LOGS = os.path.abspath("")
+
+
+def log_files_metrics(file):
+    error_count = 0
+    warning_count = 0
+
+    with open(file, "r") as file:
+        for line in file:
+            if "ERROR" in line:
```

Comment:
Just a reminder to update this so it only checks if "ERROR" is at the beginning of the line.